__tests__/app.test.js

```diff
@@ -99,6 +98,49 @@ describe("Backend testing", () => {
         });
     });
   });
-=======
->>>>>>> main
+  describe.only("PATCH /api/articles/:article_id", () => {
+    test("status: 200, responds with the updated article", () => {
```

Comment:
I would potentially add a test that has a positive integer as a value of inc_votes